service/slice_config_webhook_validation.go

```diff
@@ -247,7 +247,16 @@ func preventUpdate(ctx context.Context, sc *controllerv1alpha1.SliceConfig) *fie
 }
 
 // validateQosProfile is a function to validate the Qos(quality of service)profile of slice
-func validateQosProfile(sliceConfig *controllerv1alpha1.SliceConfig) *field.Error {
+func validateQosProfile(ctx context.Context, sliceConfig *controllerv1alpha1.SliceConfig) *field.Error {
+	if len(sliceConfig.Spec.StandardQosProfileName) > 0 && len(sliceConfig.Spec.QosProfileDetails.QueueType) > 0 {
```

Comment:
Are we doing `len()` function for string `StandardQosProfileName`? We can check the blank string "", right?